torch/csrc/api/include/torch/optim/optimizer.h

```diff
@@ -199,8 +199,9 @@ TORCH_API serialize::InputArchive& operator>>(
 /// according to the concrete optimization algorithm.
 class Optimizer : public detail::OptimizerBase {
  public:
-  using detail::OptimizerBase::OptimizerBase;
-  virtual void step() = 0;
+   using LossClosure = std::function<Tensor()>;
```

Comment:
maybe we can add the comment above: `/// A loss function closure, which is expected to return the loss value.`